src/components/TodoItem/TodoStatus/TodoStatus.tsx

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+
+interface Props {
+  checked: boolean;
+  onChangeStatus: () => void;
+}
+
+export const TodoStatus: React.FC<Props> = React.memo(({
+  checked,
+  onChangeStatus,
+}) => {
+  return (
+    <label className="todo__status-label">
+      <input
+        type="checkbox"
+        className="todo__status"
+        checked={checked}
+        onChange={onChangeStatus}
```

Comment:
Pass the checked state to the onChangeStatus function using an inline function: onChange={(e) => onChangeStatus(e.target.checked)}